src/RecoveryServices/RecoveryServices.Test/RecoveryServices.Test.csproj

```
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.0-preview" />
```

Need to use exact same version of SDK across the solution

```suggestion
    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.1-preview" />
```